README.md

```diff
@@ -28,16 +28,16 @@ Microsoft PowerToys is a set of utilities for power users to tune and streamline
 
```

Comment:
We need to add links to "current utilities" line 19 to 23

Reply:
Added them.